### ISSUE TITLE
Eigen vectors of dynamic size don't work properly

### ISSUE THREAD

**Issue by tehrengruber** [tehrengruber/LLDB-Eigen-Data-Formatter#1]
```
#include <Eigen/Dense>
#include <iostream>

int main() {
  Eigen::VectorXd a(3, 1);

  std::cout << a << std::endl;
}
```

Eigen vectors of dynamic size won't work if `.rows()` and `.cols()` are inlined. In order to make this work we need to infer the number of cols from the template parameter and get the number of rows from `m_storage.m_rows`

**Comment by randi120** [comment]
see PR #2

**Comment by tehrengruber** [comment]
Implemented by 7a07dc1b3c3e209033626b5e4bda8a6da827131c
